prog/vm/github_runner.rb

```diff
@@ -54,9 +54,16 @@ def self.pick_vm(label, project)
         amount: 1
       )
 
+      # :nocov:
+      puts "Pool is used for #{label}" unless Config.test?
```

Comment:
No need to use `:nocov:`, and disable for testing. You can catch stdout like this
https://github.com/ubicloud/ubicloud/blob/645da0c29ba6bcf020fb4e34a2e309e83277f9e7/spec/prog/vnet/subnet_nexus_spec.rb#L369